packages/react/tsconfig.json

```diff
@@ -5,7 +5,11 @@
     "target": "ESNext",
     "esModuleInterop": true,
     "moduleResolution": "node",
+    "declaration": true,
```

Comment:
I'm a bit of a ts noob what is `declaration` doing and what does the `include` prop do?

Reply:
I’m somewhat new too, but this resolved an issue where `/types.ts` was not being moved to `/dist`, preventing it from being possible to `import` the types outside of the component.

I was referencing this issue, and did a copy paste until things worked:
https://github.com/developit/microbundle/issues/58

Reply:
Weird I can see the delcaration files in my build wihtout having that option turned on

Reply:
The declaration files appear, but `types.ts` is not imported for some reason, unless you use the `include`. I needed access to `types.ts` in order to define the props for my custom `Embed` component.